app/src/main/res/layout/fragment_detailed_item.xml

```diff
@@ -225,9 +225,18 @@
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
                 android:layout_gravity="fill_horizontal|top"
-                android:contentDescription="@string/share"
                 android:backgroundTint="@color/themeColor"
+                android:contentDescription="@string/share"
                 app:srcCompat="?attr/actionModeShareDrawable" />
+
+            <com.google.android.material.floatingactionbutton.FloatingActionButton
+                android:id="@+id/locateButton"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_gravity="end|bottom"
+                android:layout_marginTop="16dp"
+                android:contentDescription="@string/locate_item"
+                app:srcCompat="@drawable/baseline_search_black_24dp" />
```

Comment:
We could use the same style for both buttons. Also it would be better if the icon was white too.
```suggestion
                android:backgroundTint="@color/themeColor"
                android:layout_marginTop="16dp"
                android:contentDescription="@string/share"
                app:srcCompat="?attr/actionModeShareDrawable" />

            <Space
                android:layout_width="10dp"
                android:layout_height="wrap_content" />

            <com.google.android.material.floatingactionbutton.FloatingActionButton
                android:id="@+id/locateButton"
                android:layout_width="match_parent"
                android:layout_height="wrap_content"
                android:layout_gravity="end|bottom"
                android:backgroundTint="@color/themeColor"
                android:layout_marginTop="16dp"
                android:contentDescription="@string/locate_item"
                app:srcCompat="@drawable/baseline_search_black_24dp" />
```